swift-ios/Task Tracker/Models.swift

```diff
@@ -35,12 +46,13 @@ enum TaskStatus: String {
 class Task: Object {
     @objc dynamic var _id: ObjectId = ObjectId.generate()
     @objc dynamic var _partition: ProjectId = ""
+    @objc dynamic var assignee: User?
```

Comment:
should i add assignee to android as well? I previously thought that was out of scope, but if that's changed I can always add it

Reply:
No, Android works a bit differently and treats optional and non-existent fields as the same, meaning we can have this in the schema and not necessarily add it to the model.